specification/components/r4/schemas/queryParameters/Identifier.yaml

```diff
@@ -0,0 +1,25 @@
+name: identifier
+in: query   
+description: |
+  The UBRN which identifies the referral or advice and guidance.
```

Comment:
Changed to 

> The UBRN which identifies the `ServiceRequest` resource.

Reply:
In retrospect, I've changed this to

> The UBRN used to search for `ServiceRequest` resources.

to avoid saying a ubrn _identifies_ a service request.